File: tests/python/test_event.py
import os,sys
sys.path.insert(0,os.path.join(os.path.dirname(os.path.abspath(__file__)),"../../python"))
#print os.path.join(os.path.dirname(os.path.abspath(__file__)),"../../python")
import chargebee
from chargebee.main import Environment
Environment.chargebee_domain="localcb.in:8080"
Environment.protocol = "http"
chargebee.ChargeBee.verify_ca_certs=False

##Copy code from api docs
chargebee.configure("test___dev__yuYlT1jLI4AXhwZpF3cuGModbc127lfOw","mannar-test")

f = open("./sample_event.json","r")
content = f.read()
print content

event = chargebee.Event.deserialize(content)
print event.id

File: tests/ruby/test_event.rb
require '../../ruby/lib/chargebee.rb';
$CHARGEBEE_DOMAIN = "localcb.in:8080"
#$CHARGEBEE_DOMAIN = "stagingcb.com"
ChargeBee.verify_ca_certs=(false)
#$ENV_PROTOCOL = "http"
#ChargeBee.verify_ca_certs=(true)

#Code from apidocs
ChargeBee.configure(:site => "mannar-test",
  :api_key => "test___dev__qIOf7uXcI3vsR3DNNOelTWmF70W1yIUR")


file = File.new("./sample_event.json")
contents = file.read
#puts contents

event = ChargeBee::Event.deserialize(contents)
puts event
puts event.event_type
